src/plugins/myItems/myItemsInterceptor.js

```diff
@@ -22,11 +22,11 @@
 
 import { MY_ITEMS_KEY } from "./createMyItemsIdentifier";
 
-function myItemsInterceptor(identifierObject, openmct) {
+function myItemsInterceptor(identifierObject, name, openmct) {
 
     const myItemsModel = {
         identifier: identifierObject,
-        "name": "My Items",
+        name,
```

Comment:
to be consistent can we convert all keys to be not a string?
eg: `"type":` -> `type` etc

Reply:
I don't know why you hate strings so much but ok... : )